### PR TITLE
Update Remove-PnPList.md

### PR DESCRIPTION
**Pull request by pagejustin**
Documenting the error upon attempting to delete a list exceeding a list view threshold

### CODE REVIEW

File: sharepoint/sharepoint-ps/sharepoint-pnp/Remove-PnPList.md [hunk 1]
```diff
@@ -14,7 +14,10 @@ online version: https://pnp.github.io/powershell/cmdlets/Remove-PnPList.html
 > [!TIP]
 > We encourage you to make improvements to this documentation. Please navigate to https://github.com/pnp/powershell/blob/dev/documentation/Remove-PnPList.md to change this file.
 
-Deletes a list
+Remove-PnPList deletes a list.
+
+###Exceeding the list view threshold
```
Comment (tseward) [inline sharepoint/sharepoint-ps/sharepoint-pnp/Remove-PnPList.md:19]: You'll need to remove the ### per PlatyPS. You can likely just use ## or > [!TIP] etc.

**Review by tseward — APPROVED** [review]

**Review by pagejustin — COMMENTED** [review]
I hope a Note is an acceptable change

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@dariomws  @get-itips

**Review by dariomws — APPROVED** [review]

**Comment by yogkumgit** [comment]
@tiburd Please merge this PR. Thanks

**Comment by yogkumgit** [comment]
@get-itips and @JohanFreelancer9

**Pull request closed without merging**